src/platform/datapath_epoll.c

```diff
@@ -33,8 +33,9 @@ CXPLAT_STATIC_ASSERT((SIZEOF_STRUCT_MEMBER(QUIC_BUFFER, Buffer) == sizeof(void*)
 
 //
 // The maximum single buffer size for coalesced IO payloads.
+// Payload size: 65535 - 8 (UDP header) - 20 (IP header) = 65507 bytes.
```

Comment:
What happens for IPv6? Its header size is bigger. We can use this for the pool allocation size, but should we be returning a different max size to actually _use_ when doing IPv6?

Reply:
Refer to this article
[(https://blog.cloudflare.com/virtual-networking-101-understanding-tap/)]
Payload size: 65507 bytes for IPv4(without options and padding field) and 65527 for IPv6(Payload length field no include P header)
Perhaps a more rigorous approach is needed here